content/normal-modal-logic/completeness/modalities-ccs.tex

```diff
@@ -178,15 +178,15 @@
 
 \begin{lem}\ollabel{lem:box-iff-diamond}
   Suppose $\Gamma$ and $\Delta$ are complete
-  $\Sigma$-consistent. Then: $\Box^{-1}\Gamma \subseteq \Delta$ if and
+  $\Sigma$-consistent. Then $\Box^{-1}\Gamma \subseteq \Delta$ if and
   only if $\Diamond\Delta \subseteq \Gamma$.
 \end{lem}
 
 \begin{proof}
   ``Only if'' direction: Assume $\Box^{-1}\Gamma \subseteq \Delta$ and
   suppose $\Diamond!A \in \Diamond\Delta$ (i.e., $!A \in \Delta$).  In
-  order to show $\Diamond!A \in \Gamma$ it suffices to show
-  $\Box\lnot!A \notin \Gamma$ for then by maximality $\lnot\Box\lnot
+  order to show $\Diamond!A \in \Gamma$, it suffices to show
+  $\Box\lnot!A \notin \Gamma$, for then by maximality, $\lnot\Box\lnot
```

Comment:
I found the wording of this sentence to be confusing, especially the "for then by maximality" part, but I wasn't sure how to rearrange it so I just split it up with commas.